repose-aggregator/components/services/container-configuration-service/impl/src/test/scala/org/openrepose/nodeservice/containerconfiguration/ContainerConfigurationServiceImplTest.scala

```diff
@@ -0,0 +1,272 @@
+/*
+ * _=_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=
+ * Repose
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Copyright (C) 2010 - 2015 Rackspace US, Inc.
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
+ */
+package org.openrepose.nodeservice.containerconfiguration
+
+import java.net.URL
+
+import org.junit.runner.RunWith
+import org.mockito.Mockito.{never, verify, when}
+import org.mockito.{Matchers => MockitoMatchers}
+import org.openrepose.commons.config.manager.UpdateListener
+import org.openrepose.core.container.config._
+import org.openrepose.core.services.config.ConfigurationService
+import org.scalatest.junit.JUnitRunner
+import org.scalatest.mock.MockitoSugar
+import org.scalatest.{BeforeAndAfterEach, FunSpec, Matchers}
+
+@RunWith(classOf[JUnitRunner])
+class ContainerConfigurationServiceImplTest extends FunSpec with Matchers with MockitoSugar with BeforeAndAfterEach {
+
+  final val DefaultClusterId = "defaultClusterId"
```

Comment:
Since we have a mixed code base, there's an argument to be made the being consistent across our codebase is more important than following the Scala style guide. If you want to use the Java constant form, I wouldn't fight it.